frontend/src/components/parentData/ParentBuild.js

```diff
@@ -32,9 +32,9 @@ const ParentSeries = () => {
         fetchData();
 
         // returned function will be called on component unmount
-        return () => {
-            dispatch({ type: 'flushParentData' });
-        };
+        //return () => {
```

Comment:
why has this been removed?

Reply:
Originally faced some load time issues and removing the flushing fixed it as weird as it sounds. The load was stuck in a loop which might have been caused by the dispatch method. I will try removing this change.